randovania/games/samus_returns/logic_database/Area 2 - Dam Interior.txt

```diff
@@ -625,14 +704,20 @@ Extra - asset_id: collision_camera022
           Any of the following:
               Space Jump
               Spider Boost (Intermediate) and Can Spider Boost
+              Bomb and Infinite Bomb Jump (Intermediate)
               All of the following:
                   Spider Ball
-                  High Jump Boots or Wall Jump (Intermediate)
-              Bomb and Infinite Bomb Jump (Intermediate)
+                  Any of the following:
+                      High Jump Boots or Super Jump (Intermediate) or Unmorph Extend (Intermediate)
+                      # Jump over the Black Wall Fires
+                      Wall Jump (Advanced)
+                      # Defeat the Black Wall Fires
+                      Defeat Non-Counterable Enemies
               All of the following:
-                  Unmorph Extend (Intermediate)
-                  High Jump Boots or Wall Jump (Intermediate)
-              High Jump Boots and Super Jump (Intermediate)
+                  High Jump Boots
+                  Single-wall Wall Jump (Intermediate) or Super Jump (Intermediate) or Unmorph Extend (Intermediate)
+              # Jump over the Black Wall Fires
+              Single-wall Wall Jump (Intermediate) and Wall Jump (Advanced)
```

Comment:
slightly confused at this. doesnt this still need HJ?

Reply:
Added a video